rhoam-manifests/0.5.0-manifest.txt

```diff
@@ -0,0 +1,1966 @@
+services-rhoam:0.5.0/google.golang.org/appengine-v1.5.0
+services-rhoam:0.5.0/google.golang.org/appengine-v1.6.1
+services-rhoam:0.5.0/google.golang.org/appengine-v1.6.2
+services-rhoam:0.5.0/google.golang.org/appengine-v1.6.3
+services-rhoam:0.5.0/google.golang.org/appengine-v1.6.5
+services-rhoam:0.5.0/google.golang.org/appengine-v1.6.6
+services-rhoam:0.5.0/google.golang.org/cloud-v0.0.0-20151119220103-975617b05ea8
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20180817151627-c66870c02cf8
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20180831171423-11092d34479b
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20181029155118-b69ba1387ce2
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20181202183823-bd91e49a0898
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190122154452-ba6ebe99b011
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190307195333-5fe7a883aa19
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190404172233-64821d5d2107
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190418145605-e7d98fc518a7
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190425155659-357c62f0e4bb
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190502173448-54afdca5d873
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190530194941-fb225487d101
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190716160619-c506a9f90610
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190801165951-fa694d86fc64
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190819201941-24fa4b261c55
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190911173649-1774047e7e51
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190925194540-b8fbc687dcfb
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20190927181202-20e1ac93f88c
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20191108220845-16a3f7862a1a
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20191115194625-c23dd37a84c9
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20191216164720-4f79533eabd1
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20191230161307-f3c370f40bfb
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200108215221-bd8f9a0ef82f
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200115191322-ca5a22157cba
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200122232147-0452cf42e150
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200204135345-fa8e72b47b90
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200212174721-66ed5ce911ce
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200224152610-e50cd9704f63
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200331122359-1ee6d9798940
+services-rhoam:0.5.0/google.golang.org/genproto-v0.0.0-20200420144010-e5e8543f8aeb
+services-rhoam:0.5.0/google.golang.org/grpc-v0.0.0-20160317175043-d3ddb4469d5a
+services-rhoam:0.5.0/google.golang.org/grpc-v1.14.0
```

Comment:
When validating this file I used the `google.golang.org/grpc` as a random example. The grpc-v1.14.0 is not in my installation.  I am wondering what we would need to do in this case. To help resolve this issue my installed go version is `go1.13.8`

Reply:
I think as the manifest aims to run vulnerability checks on dependencies used, we are safe with whichever official Go version we are using.

Reply:
Looking into this a bit more I feel the first time I tried this I may have done something wrong and this should be regraded as an none issue.